test/extended/cli/explain.go

```diff
@@ -566,7 +556,7 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 				e2e.Logf("Checking %s...", bt)
 				exist, err := exutil.DoesApiResourceExist(oc.AdminConfig(), bt.Resource, groupName)
 				o.Expect(err).NotTo(o.HaveOccurred())
-				if !exist {
+				if !exist && groupName == "security.openshift.io" {
```

Comment:
Skipping this group excludes three resources from testing:
- `podsecuritypolicyreviews`
- `podsecuritypolicyselfsubjectreviews`
- `podsecuritypolicysubjectreviews`

Reply:
The change only skips if a type within the group does not exist, right? Isn't that exactly what we want in this case, since OpenShift includes more types than MicroShift?

I'm surprised to see the groupName test in the condition, though. @chiragkyal , why is that there?

Reply:
The change will skip the entire group if any resource (type) within that group does not exist. This test expects that if a group is present, all of its resources must also be present and that is where the problem lies because for MicroShift we have only `securitycontextconstraints` under `"security.openshift.io"`. So, if we try to skip any resource, it will skip the entire apigroup. Checking for each resource in each apigroup is something has not been done so far. See [here](https://github.com/openshift/origin/pull/27615#discussion_r1180417677) and [here](https://redhat-internal.slack.com/archives/C03DP9PABNC/p1684999750574689) 


As we are having problem only with `"security.openshift.io"` group, I thought to skip this group from here and test it separately under a [new test ](https://github.com/openshift/origin/pull/27897/files#diff-f0e9098abef3f083227a769d2b891489aec36b40e6948db84b464d032c359ed2R602) with `microShiftTypes` but it will again cause problem for OpenShift with partial testing of this group. It's a deadlock situation.